src/test/java/gov/di_ipv_fraud/pages/FraudPageObject.java

```diff
@@ -124,11 +124,10 @@ public void navigateToIPVCoreStub() {
         String coreStubUsername = configurationService.getCoreStubUsername();
         String coreStubPassword = configurationService.getCoreStubPassword();
         String coreStubUrl = configurationService.getCoreStubUrl();
-        Driver.get()
-                .get("https://" + coreStubUsername + ":" + coreStubPassword + "@" + coreStubUrl);
-        waitForTextToAppear(IPV_CORE_STUB);
+       Driver.get()
+                .get("http://" + coreStubUsername + ":" + coreStubPassword + "@" + coreStubUrl);
```

Comment:
Should this not be https?

Reply:
This was the simplest workaround for running the tests against a local stub, where a HTTP connection will fail (any provided username/password is ignored).
HTTP was checked to be auto promoted to HTTPS when running against the core-stub on the environments.

The alternative is to add a boolean environment var for HTTP/HTTPS or build up the full URL in an environment variable and provide it here?

Reply:
Yeah I would prefer if we could do this with a boolean via a passed in argument rather than relying on the browser being clever enough to switch it